Exercise -1 - database.txt

```diff
@@ -0,0 +1,18 @@
+create database employee;
+
+CREATE TABLE department (
+  department_id INTEGER PRIMARY KEY,
+  department_name VARCHAR(30) NOT NULL,
+);
+CREATE TABLE employee (
+  employee_id INTEGER PRIMARY KEY,
+  employee_name VARCHAR(30) NOT NULL,
+  salary INTEGER,
+  department_id,
```

Comment:
what is the data type for the department?
Did you run the statement to see if it works before submitting?